### PR TITLE
Legion hearts renamed and reskinned to legion's soul

### PR DESCRIPTION
**Pull request by coiax**
Also readded the statistical tracking that got taken out during the
dismemberment merge, I assume accidentally.
:cl: coiax, OPDingo
tweak: Renamed and reskinned legion's heart to legion's soul to avoid confusion; it is not a substitute heart, it is a supplemental chest organ.
rscadd: Legion's souls are now more obviously inert when becoming inert.
/:cl:
image

### CODE REVIEW

**Comment by Armhulen** [comment]
I'm fairly sure inert legion hearts still work as hearts, right? does this mean a nonbeating heart will still work?

**Comment by coiax** [comment]
Legion hearts, despite appearances, are not actually hearts.

**Comment by Aloraydrel** [comment]
I dont think this is canon

**Comment by Supermichael777** [comment]
you just jam them in the dudes chest-hole and he starts getting ~~hunted by Daleks~~ better.

**Comment by Armhulen** [comment]
ah shit
I removed someones heart and replaced it with a fake one

FUCK

**Comment by Supermichael777** [comment]
making it a heart should be as simple as swaping the paths to:
 /obj/item/organ/heart/hivelord_core/legion
So you realy can get the heart of the damned

**Comment by phil235** [comment]
Naming the thing "heart" when it's not a heart was a mistake.

**Comment by coiax** [comment]
@KorPhaeron FIX MUH LORE PLZ. Call it a legion's soul?

**Comment by Armhulen** [comment]
If I kill a legion and the dead body is ginger does that mean I won't get its soul?

File: code/modules/mob/living/simple_animal/hostile/mining_mobs.dm [hunk 1]
```diff
@@ -248,10 +248,15 @@
 	..()
 	spawn(2400)
 		if(!owner && !preserved)
-			inert = 1
-			desc = "The remains of a hivelord that have become useless, having been left alone too long after being harvested."
+			go_inert()
 		else
-			preserved = 1
+			preserved = TRUE
+			feedback_add_details("hivelord_core", "[src.type]|implanted")
```
Comment (KorPhaeron) [inline code/modules/mob/living/simple_animal/hostile/mining_mobs.dm]: What is this feedback for?

Comment (coiax) [inline code/modules/mob/living/simple_animal/hostile/mining_mobs.dm]: It indicates that the legion heart has been implanted in a person, and been stabilised by merit of being implanted.

**Comment by optimumtact** [comment]
still awaiting sprites?

**Comment by coiax** [comment]
I have sprites, just need to actually finish this.

**Pull request merged**